ansible/roles/sysprep/tasks/main.yml

```diff
@@ -195,3 +188,12 @@
     name: kubelet
     enabled: true
     state: started
+
+- name: Disable Password Authentication
```

Comment:
I _think_ this is ok since its in the `sysprep` role. 
Do we want to be more defensive and add `when: packer_builder_type.startswith('vsphere') or packer_builder_type.startswith('vmware')`?

Reply:
Also thought about this.

I think this is a best practices sort of thing to disable password auth. If I understand correctly, CAPI providers leverage cloud-init to configure hosts. I think CAPPP uses ssh-key authentication